README.md

```diff
@@ -225,6 +226,10 @@ However, this markup can be simplified using a simple CSS class to avoid repetit
   <use xlink:href="path/to/dist/feather-sprite.svg#circle"/>
 </svg>
 ```
+### Figma
+Feather icons are available as a component library in Figma. A master set of the icons is maintained [here](https://www.figma.com/file/dyJRSFTIajik4cdkcXN8yA3K/Feather-Icons-Set?node-id=0%3A1).
+
+To use simply login to your Figma account and **Duplicate** the file to your Drafts.
```

Comment:
This is great! I usually try to avoid links that just say "here" whenever possible. Also, in the documentation I try to refer to the project as "Feather" instead of "Feather icons." What about something like this:

```
Feather is available as a [Figma component library](#). To use the components, log in to your Figma account and **Duplicate** the file to your drafts.
```

We should probably also change the name of the Figma file to just "Feather."

Reply:
Completely agree! Will update this PR and Figma file.

Reply:
Updated with your suggested copy. Ready for review @colebemis